WordPressKit/Time-based data/ReferrerStatsType.swift

```diff
@@ -0,0 +1,84 @@
+public struct ReferrerStatsType {
+    public let period: StatsPeriodUnit
+    public let periodEndDate: Date
+
+    public let totalReferrerViews: Int
```

Comment:
In `ClicksStatsType`, there appears to be an analogous property called `totalClicksCount`. Is this worth renaming to `totalViewsCount` or `totalRefererredViewsCount` for consistency with that and `StatsReferrer` (`viewsCount`) below?

Reply:
Yup, absolutely! Thanks for catching that!